Cannot import ReferencePattern from markdown

This bug appeared after `markdown` was updated to `3.1.1`.

**Code that generated the error**
```bash
$ make
Traceback (most recent call last):
  File "/home/santi/.miniconda3/envs/urubu/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/santi/.miniconda3/envs/urubu/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/santi/.miniconda3/envs/urubu/lib/python3.6/site-packages/urubu/__main__.py", line 18, in <module>
    from urubu import main
  File "/home/santi/.miniconda3/envs/urubu/lib/python3.6/site-packages/urubu/main.py", line 25, in <module>
    from urubu import project
  File "/home/santi/.miniconda3/envs/urubu/lib/python3.6/site-packages/urubu/project.py", line 33, in <module>
    from urubu import readers, processors
  File "/home/santi/.miniconda3/envs/urubu/lib/python3.6/site-packages/urubu/processors.py", line 31, in <module>
    from urubu import md_extensions
  File "/home/santi/.miniconda3/envs/urubu/lib/python3.6/site-packages/urubu/md_extensions.py", line 31, in <module>
    from markdown.inlinepatterns import ReferencePattern, REFERENCE_RE, SHORT_REF_RE
ImportError: cannot import name 'ReferencePattern'
make: *** [Makefile:6: build] Error 1
```

--- a/environment.yml
+++ b/environment.yml
@@ -7,7 +7,7 @@ dependencies:
     # Install Urubu's requirements from conda
     - jinja2>=2.7
     - beautifulsoup4
-    - markdown=2.6.11
+    - markdown<3
     - pygments
     - pyyaml
     - pip:

--- a/environment.yml
+++ b/environment.yml
@@ -7,7 +7,7 @@ dependencies:
     # Install Urubu's requirements from conda
     - jinja2>=2.7
     - beautifulsoup4
-    - markdown
+    - markdown=2.6.11
     - pygments
     - pyyaml
     - pip:

